### PR TITLE
refactor: add auth code status, fix getAuthStatus result for non 200

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: src/store/modules/AuthModule.js [hunk 1]
```diff
@@ -160,30 +166,33 @@ class AuthModule extends VuexModule {
...
   }
 
   @Action
-  changeAuthStatusByCode({ code, hash }) {
-    this.setAuthByCode(code);
+  changeAuthByStatus({ status, hash }) {
+    this.updateAuthStateByStatus(status);
     const isAuthorizedNew = this.isAuthorized;
+
+    const code = STATUS_TO_CODE[status] || 401;
```
Comment (kazurus) [inline src/store/modules/AuthModule.js]: 401 => STATUS_TO_CODE.DEFAULT

File: tests/unit/specs/service/auth.spec.js [hunk 2]
```diff
@@ -133,7 +134,7 @@ describe('auth service', () => {
       const res = await authService.getAuthStatus();
 
       expect(res).toEqual({
-        status: 200,
+        status: AUTH_STATUS_CODE.LOGGED_IN,
```
Comment (kazurus) [inline tests/unit/specs/service/auth.spec.js]: maybe can we rename AUTH_STATUS_CODE to AUTH_STATUS?

File: src/service/mode.js [hunk 3]
```diff
@@ -24,7 +25,7 @@ export const validateIdentityServer = async serverUrl => {
     const respCode = e.response && e.response.status;
 
     switch (respCode) {
-      case 401:
+      case AUTH_STATUS_CODE.NOT_LOGGED:
```
Comment (kazurus) [inline src/service/mode.js]: AUTH_STATUS_CODE.NOT_LOGGED => 401?

File: tests/unit/specs/service/auth.spec.js (hunk 2 above)
Comment (dudiq) [reply tests/unit/specs/service/auth.spec.js]: it will crossed with `METHODS.AUTH_STATUS` that's why i named as `AUTH_STATUS_CODE` =\

**Pull request merged**